setup.py

```diff
@@ -12,13 +12,13 @@
 P_REPO_ROOT = Path(__file__).absolute().parents[0]
 
 
-def get_long_description():
+def get_long_description(title_only=False):
```

Comment:
nit: Should we rename this to `get_description` now that they might not be so long?

Reply:
Good idea. Done